### PR TITLE
Helm chart for multiple validators.

### PR DESCRIPTION
**Pull request by LaContra**
- Add NFS persistent storage support (tested with k3s).
- Add multiple validators support.
- Add [fallback web3provider ](https://docs.prylabs.network/docs/prysm-usage/setup-eth1/#adding-fallback-eth1-nodes) for prysm beacon node.
- Roll deployment when wallet password changes. 
- Add image base URL variable.

### CODE REVIEW

File: eth2prysm/templates/validator-deployment.yaml [hunk 1]
```diff
@@ -46,16 +51,38 @@ spec:
...
+      {{- if $.Values.nfsEnabled }}
+      securityContext:
+        runAsUser: {{ $.Values.nfsUser }}
+        runAsGroup: {{ $.Values.nfsGroup }}
+      {{- end }}
       serviceAccountName: validator
       restartPolicy: Always
+
+---
+{{- end}}
```
Comment (chingyi-lin) [inline eth2prysm/templates/validator-deployment.yaml]: 1 - `{{- end }}` . some other places also have `{{- end}}`
2 -  add EOF

File: eth2prysm/values.yaml [hunk 2]
```diff
@@ -2,21 +2,45 @@
...
 
 namespace: prysm
-imageVersion: v1.3.3
 
 ethereumTestnet: pyrmont
 
+nfsEnabled: true
+nfsServerIp: 172.20.10.10
+nfsUser: 1001
+nfsGroup: 2000
```
Comment (chingyi-lin) [inline eth2prysm/values.yaml]: per discussion, let's wrap them under `nfs`

File: eth2prysm/values.yaml [hunk 3]
```diff
@@ -2,21 +2,45 @@
 # This is a YAML-formatted file.
 
 namespace: prysm
-imageVersion: v1.3.3
 
 ethereumTestnet: pyrmont
 
+nfsEnabled: true
```
Comment (chingyi-lin) [inline eth2prysm/values.yaml]: note: comment available persistent storage options and take the string value directly

File: eth2prysm/values.yaml [hunk 4]
```diff
@@ -2,21 +2,45 @@
...
-  walletHostPath: /data/prysm/wallet
-  walletPassword: "example-password"
+validators:
+  validator1:
+    name: validator1
+    dataVolumePath: /data/prysm/validator-1
+    walletVolumePath: /data/prysm/wallet-1
+    walletPasswordSecretName: wallet-password-1
+    walletPassword: "example-password-1"
+    startWaitTime: 780
```
Comment (chingyi-lin) [inline eth2prysm/values.yaml]: as the user is likely to use the same wait time for all validators, maybe we can make it a global configuration?

File: eth2prysm/templates/validator-deployment.yaml (hunk 1 above)
Comment (LaContra) [reply eth2prysm/templates/validator-deployment.yaml]: Done.

File: eth2prysm/values.yaml (hunk 2 above)
Comment (LaContra) [reply eth2prysm/values.yaml]: Done. Also removed nfsEnabled.

File: eth2prysm/values.yaml (hunk 3 above)
Comment (LaContra) [reply eth2prysm/values.yaml]: Done.

File: eth2prysm/values.yaml (hunk 4 above)
Comment (LaContra) [reply eth2prysm/values.yaml]: Done.

**Review by chingyi-lin — APPROVED** [review]

**Pull request merged**